site/styles/_typography.scss

```diff
@@ -0,0 +1,84 @@
+@function to-rem($pixels) {
```

Comment:
I'm confused about what this is for... wouldn't it be better to fully commit to using `rem` or `px` instead of inputting one and outputting the other?

Reply:
`rem` feels abstract and unintuitive e.g. `2.8125rem; 1.4375rem; 0.875rem` even if you know what the base size is. I think its good to describe the actual value it translates/computes to. Also makes it easier if design wants to change the value.
```
@mixin heading-1-style() {
  @include typography-style(2.8125rem, 0.4px, $font-weight-normal, 8px, var(--body-color)); // 46px
```
Will change if you prefer though I don't mind

Reply:
https://stackoverflow.com/questions/30374863/why-use-rem-instead-px-when-its-the-same-anyway

Using `rem` has a major accessibility advantage for people with poor eyesight that have increased their browser `rem` base size from the default of `16px`. A function that hardcodes division by `16` is going to screw them over. Yes, I agree that the floating point `rem` values are abstract and unintuitive and I like that you tried to improve on that, but this implementation is not a good solution.

But having said all that, I'm of the opinion that anyone who has increased their browser base size is going to have a poor result with our site anyway with a mix of `rem` and `px` and a design that is trying to be pixel perfect.

Reply:
I think we should stick with all `px` for now.  There is probably value in converting things to `rem` and `em`, but it's something that we should have an elab for and it will affect more than just fonts so I reckon this PR is not the right place for it.

Reply:
@matthewthomsonnz What is the resolution on this one? We are agreeing to stick with `px` for now right, so `to-rem()` should be `rem`oved?

Reply:
@stajs Oh yep. Reverted file to before function was added. 

iseewhatyoudidthere.jpg